config/defaults.mk

```diff
@@ -42,6 +42,9 @@ STATIC = YES
 SHARED = NO
 
 # CUDA configuration options
+#
+# If you set MFEM_USE_ENZYME=YES, this has to be configured to use cuda with
+# clang as it's host compiler.
```

Comment:
This was a bit unclear to me? What is "this"? Are there specific flags that should be changed below?

Reply:
This refers to `CUDA_CXX`

Reply:
Could you clarify the comment? e.g. "the CUDA compiler given by CUDA_CXX should be configured to use clang as its host compiler"